### PR TITLE
Scan build

### PR DESCRIPTION
**Pull request by GregoryVds**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: c
 sudo: required
 dist: xenial
 install:
-    - sudo apt-get install -y cmake python3 python3-pip python3-setuptools pkg-config libcapstone-dev cppcheck curl wget check
+    - sudo apt-get install -y cmake python3 python3-pip python3-setuptools pkg-config libcapstone-dev cppcheck curl wget check clang-tools-6.0
```
Comment (matttbe) [inline .travis.yml:5]: can you not install clang-tool

EDIT: OK, just saw it was not available on xenial... and only in the "update" repo.

You can also add PPA if you want a recent version ;-)
https://apt.llvm.org/

File: CMakeLists.txt [hunk 2]
```diff
@@ -38,3 +38,9 @@ add_test(NAME cppcheck
                           --suppress=unusedStructMember ${LIBCONVERT_SOURCES}
          WORKING_DIRECTORY ${PROJECT_SOURCE_DIR})
 
+add_test(NAME scan-build
+         COMMAND bash -c "rm -rf scan-build && \
+                          mkdir scan-build && \
+                          cd scan-build && \
+                          scan-build-6.0 cmake ${PROJECT_SOURCE_DIR} && \
```
Comment (matttbe) [inline CMakeLists.txt:45]: I guess you cannot use scan-build here, right?

**Review by matttbe — APPROVED** [review]
just one question about clang-tools packages ;-)
But I guess it can be merged like that!

File: CMakeLists.txt (hunk 2 above)
Comment (GregoryVds) [reply CMakeLists.txt:45]: No, not with the Ubuntu 16 package

**Comment by GregoryVds** [comment]
Thanks for the review Matt! I will merge like this then, build passed :-)

**Pull request merged**